src/Control/Lens/At.hs

```diff
@@ -454,17 +458,25 @@ instance At (Maybe a) where
   {-# INLINE at #-}
 
 instance At (IntMap a) where
+#if MIN_VERSION_containers(0,5,8)
```

Comment:
It sounds like there are plans (https://github.com/haskell/containers/issues/325) to add `RULES` for `Data.IntMap.alterF`, so I'll opt to keep this definition, then.